addon/components/ember-tbody/component.js

```diff
@@ -306,6 +309,11 @@ export default Component.extend({
     this.collapseTree.set('rowMetaCache', this.rowMetaCache);
     this.collapseTree.set('rows', rows);
 
+    run.schedule('actions', () => {
+      // eslint-disable-next-line ember-best-practices/no-side-effect-cp
+      this.set('dataTestRowCount', this.get('wrappedRows.length')); // eslint-disable-line ember/no-side-effects
```

Comment:
do we need two `eslint-disable` comments here?

Reply:
There are indeed two rules which error on this.

It is actually a bit annoying since the side effects really begin on line 309, but I presume the lint only catches on `this.set`